src/main/resources/db/changelog/001-create-table-wallets.yaml

```diff
@@ -23,15 +23,16 @@ databaseChangeLog:
                 -  column:
                      name: id
                      type: bigint
-#                     autoIncrement:  true
+                     autoIncrement:  true
                      constraints:
                        primaryKey: true
                        nullable: false
                 -  column:
-                     name: firstname
+                     name: name
                      type: varchar(50)
                 -  column:
-                     name: lastname
-                     type: varchar(50)
+                     name: creation_date
```

Comment:
One column cannot have 2 names and 2 types, I think that is fault caused by merging